components/comment/video/modal/modal.ts

```diff
@@ -75,7 +75,7 @@ export default class AppCommentVideoModal extends BaseModal {
 			return false;
 		} else if (this.video.comment.user.id === this.app.user.id) {
 			return false;
-		} else if (this.video.game.developer.id === this.app.user.id) {
+		} else if (this.video.game.hasPerms()) {
```

Comment:
i think we should still let collaborators vote, otherwise old votes will not be togglable